x86_info_term.py

```diff
@@ -805,14 +810,17 @@ def download_data(args):
 
     for dataset in DATASETS:
         # Download file
-        url = '%s/%s' % (dataset.base_url, dataset.path)
-        print('Downloading %s...' % url)
-        with urllib.request.urlopen(url) as f:
-            data = f.read()
-
-        # Write output file
-        with open(dataset.full_path, 'wb') as f:
-            f.write(data)
+        if (os.path.exists(dataset.full_path)):
```

Comment:
Don't re-download files if they already exist. This is useful when trying to repeatedly build the cache because as it is now, if the files download but the cache build fails, you'll download the files every time.